app/controllers/conference_controller.rb

```diff
@@ -13,19 +13,34 @@ def show; end
 
   def schedule
     @rooms = @conference.venue.rooms if @conference.venue
+    unless @conference.program.events.scheduled.any?
+      redirect_to events_conference_path(@conference.short_title)
+    end
+
     @events = @conference.program.events
-    @events_xml = @events.scheduled.order(start_time: :asc).group_by{ |event| event.start_time.to_date }
+    @events_xml = @events.scheduled.group_by{ |event| event.start_time.to_date }
```

Comment:
I'd rather we did not change that. I saw you included it in the definition of scheduled, but if for some reason that definition changes in the future, the XML export will be buggy. For the XML export the order is of high importance.